bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/ButtonImpl.java

```diff
@@ -114,6 +115,11 @@ public Map<String, Object> getProperties() {
         return properties;
     }
 
+    @Override
```

Comment:
This method is already implemented in `AbstractBaseImpl` if `type` property is not present it will return null, why need to override

Reply:
Because for button the type property is not present in schema, but the model.json created for it has "type", so I had to override here.